ui/src/api/index.ts

```diff
@@ -25,6 +26,7 @@ Hydra.resources.factory.addMixin(ProjectMixin)
 Hydra.resources.factory.addMixin(CSVMappingMixin)
 Hydra.resources.factory.addMixin(CSVSourceCollectionMixin)
 Hydra.resources.factory.addMixin(CSVSourceMixin)
+Hydra.resources.factory.addMixin(CSVColumnMixin)
```

Comment:
As this keeps growing, you might combine all mixins in a single module and add them all at once

```js
import * as Mixins from './mixins'

Hydra.resources.factory.addMixin(...Object.values(Mixins))
```

Reply:
Doesn't that mean I still have to list all the mixins in `mixins/index.ts`?
And TypeScript is not too happy about this, as it doesn't know the type of the values of `Mixins`.

Reply:
Yes, would still have to `export` from `index.ts`.

It's just a suggestion. Feel free to ignore :)

Reply:
I'll keep it here for now, but will probably extract at some point